bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
```

Comment:
namespace `mysql` will be not created, so the script will fail in attempt to apply stateful set